### DISCUSSION TITLE
multiple builds in a mono repository (external builds using trace-command and database upload)

### DISCUSSION THREAD

**Discussion by HubDevUser** [github/codeql#5870] [Q&A] [answered]
We currently run Semmle enterprise. The code repository is monolithic. It has multiple CI pipelines. Each pipeline builds a separate section of code for different services. Creating a database and running codeql tracing on each commit for each service causes overwriting of the scanned code from other services builds. Has this scenario been considered when using codeql\Semmle I.e. multiple builds of the same language in the same repository? are there any suggestions on how to address this problem? Perhaps create projects in Semmle for each service using the same repository root URL?

**Comment by hvitved** [comment, marked as answer]
Hi

We are working towards better support for monorepos in CodeScanning (I don't know the exact timeline), but that feature is not planned for LGTM Enterprise.

Tom

**Comment by sam-cha-tfs** [comment]
Has this been addressed? We are running into the same issue.
